Packs/CommonPlaybooks/Playbooks/playbook-CVE_Enrichment_-_Generic_v2.yml

```diff
@@ -172,13 +175,13 @@ tasks:
     isautoswitchedtoquietmode: false
   "5":
     id: "5"
-    taskid: 8f3572d2-dbfc-4c4b-82af-94d06b99e727
+    taskid: e2170ae3-622d-45d9-882b-b1ee14f1e1b3
     type: condition
     task:
-      id: 8f3572d2-dbfc-4c4b-82af-94d06b99e727
+      id: e2170ae3-622d-45d9-882b-b1ee14f1e1b3
       version: -1
-      name: Is "CVE Search" integration enabled?
-      description: Verify that there's a valid instance of "CVE Search" enabled.
+      name: Is CIRCL CVE Search integration enabled?
```

Comment:
no need for this integration check. `!cve` is a generic command.

Reply:
the !cve command is not generic, if the CIRCL CVE Search integration is not enabled the command fails. I changed the command used within task number 6 to 'cve (CIRCL CVE Search)'